tools/proteomiqon_joinquantpepionswithproteins/.shed.yml

```diff
@@ -0,0 +1,11 @@
+categories:
+    - Proteomics
+description: The tool JoinQuantPepIonsWithProteins combines results from ProteinInference and PSMBasedQuantification.
```

Comment:
mh, here is a `name` or so missing isn't it? The repo name "proteomiqon_joinquantpepionswithproteins"?

Reply:
Yeah! I have no idea how that happened 😄

Reply:
Fixed, thank you!